.github/workflows/publish-docker-images.yml

```diff
@@ -73,6 +73,14 @@ jobs:
           roleId: ${{ secrets.VAULT_ROLE_ID }}
           secretId: ${{ secrets.VAULT_SECRET_ID }}
 
+      - uses: elastic/apm-pipeline-library/.github/actions/docker-login@current
+        with:
+          registry: ""
```

Comment:
what's the reason for being empty?

Reply:
https://github.com/elastic/opbeans-node/blob/70da0522a03b365ad44d4d576d74c698e0ee684c/.github/workflows/release.yml#L21-L22 uses the registry explicitly

Reply:
Ah ok,

I tried `index.docker.io/v1/`  which did not work.

Then I tried `""` because it then uses the default registry, which is dockerhub.

Reply:
https://github.com/elastic/apm-pipeline-library/pull/2125/commits/037d34baf05dd0130e4e3a77da74dd6e7c38298a